### ISSUE TITLE
error

### ISSUE THREAD

**Issue by cjmicro** [akosidencio/Madmimi-Joomla#1]
i installed this and it did add the user to the madmimi list, but returns an error:
 Fatal error: Cannot redeclare class MadMimi in /home/xxxxx/public_html/plugins/user/jomimi/madmimi-api/MadMimi.class.php on line 37
Please advise.

**Comment by akosidencio** [comment]
may I know your website url so that I can properly check the errors?

thanks

**Comment by cjmicro** [comment]
I had to unpublish it because it would  cause the error when people register

**Comment by akosidencio** [comment]
yes I understand. but if want to proceed and use the plugin, maybe I can 
help you to fix the problem

**Comment by cjmicro** [comment]
Retiredracehorseproject.org.   I will publish it right now

**Comment by akosidencio** [comment]
Are you using a third party registration module?

**Comment by akosidencio** [comment]
I saw your site and it seems you are using community builder as your 
registration module. The plugin is works only with joomla default 
registration module not yet with other third party module such as 
community builder.

thanks

**Comment by cjmicro** [comment]
Community builder subscription plugin

**Comment by cjmicro** [comment]
OK thank you
